org.gnome.World.Secrets.json

```diff
@@ -0,0 +1,122 @@
+{
+    "app-id": "org.gnome.World.Secrets",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "secrets",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/cmake",
+        "/lib/debug",
```

Comment:
```suggestion
```
it's actually needed and will be in a separate flatpak.

Reply:
Thanks, didn't know.